### DISCUSSION TITLE
开启 `禁止跳转动态详情` 后 `详情 >` 允许右键菜单打开

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1416] [功能建议 / Ideas]
## **关于哪一项功能**
禁止跳转动态详情

## **期望的效果**
支持右键菜单打开，和 #593 一样。有中键点击打开的需求。

**Comment by the1812** [comment]
确定一定是打开至这个动态的详情吗, 我可以拿到动态 id 拼一个链接出来

**Reply by the1812** [reply]
不对啊 被转发的动态就跳到原动态去了
要不你就按住 ctrl 点它也是后台打开(

**Reply by Geeyun-JY3** [reply]
> 要不你就按住 ctrl 点它也是后台打开(

有时候纯鼠标党（

**Reply by Geeyun-JY3** [reply]
对哦这个 `详情 >` 只会在被转发的原动态有。

所以这个 feature 能实现吗？

**Reply by the1812** [reply]
我最近在准备 #1208 的接口, 扩充一些被转发的信息, 或许能把这个实现了
